composer.json

```
@@ -12,11 +12,11 @@
         }
     ],
     "require": {
-        "php": "~8.1.0 | ~8.2.0",
-        "illuminate/console": "^9.0 | ^10.0",
-        "illuminate/http": "^9.0 | ^10.0",
-        "illuminate/support": "^9.0 | ^10.0",
-        "livewire/livewire": "^2.0 | ^3.0"
+        "php": "~8.1.0 | ~8.2.0 | ~8.3.0",
```

8.1 is niet meer supported bij laravel 11
```suggestion
        "php": "8.2.* | 8.3.*",
```